src/PyCall.jl

```diff
@@ -115,7 +117,7 @@ function pyincref_(o::PyPtr)
 end
 
 function pyincref(o::PyObject)
-    pyincref_(o.o)
+    pyincref_(PyPtr(o))
```

Comment:
849adab01aae00484615060ae66b07889fbbb858 will change these slightly…

Reply:
I see. But is there something I should do in this PR?